src/axom/quest/examples/quest_signed_distance_interface.cpp

```diff
@@ -274,12 +276,16 @@ void generate_uniform_box_mesh( mint::UniformMesh*& mesh, Arguments& args)
     hi = mesh_box_max;
   }
 
-
-  SLIC_INFO( "box min: [" << lo[0] << " " << lo[1] << " " << lo[2] << "]" );
-  SLIC_INFO( "box max: [" << hi[0] << " " << hi[1] << " " << hi[2] << "]" );
-  SLIC_INFO( "constructing Uniform Mesh: [" << args.box_dims[0] <<
-             " " << args.box_dims[ 1 ] <<
-             " " << args.box_dims[ 2 ] << "]" );
+  //output some information
+  {
+    const primal::Point<double,3> lowerPoint(lo, 3);
+    const primal::Point<double,3> upperPoint(hi, 3);
+    auto bbox = primal::BoundingBox<double,3>(lowerPoint, upperPoint);
+    SLIC_INFO( "bounding box " << bbox );
+    
+    const primal::Point<int,3> bdims(args.box_dims.data(), 3);
+    SLIC_INFO( "constructing Uniform Mesh of resolution " << bdims );
+  }
```

Comment:
Is this needed? 

You are constructing primal objects just so that they get printed?

Reply:
Yes -- the output is significantly better from the primal objects.

Specifically, the Bounding box output provides the range of values.
E.g.:
```
{ min:  (0.03,  -2.1684e-19,-2.1684e-19); 
  max:  (0.0335,0.03, 0.03); 
  range:<0.0035,0.03, 0.03> }
```

This helped me (the user in this case) determine what bounding box I should provide to the code to get the desired results.